api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreProperties.java

```diff
@@ -65,4 +67,102 @@ public BiometricEngine getBiometricEngine() {
+        PixErrorHandlingService handler = null;
+        if (isPropertySet(propertyName)) {
+            handler = getComponentByPropertyName(propertyName, PixErrorHandlingService.class);
+        }
+        return handler;
+    }
+    
+    public PdqErrorHandlingService getPdqErrorHandlingService() {
+        String propertyName = RegistrationCoreConstants.GP_MPI_PDQ_ERROR_HANDLER_IMPLEMENTATION;
+        PdqErrorHandlingService handler = null;
+        if (isPropertySet(propertyName)) {
+            handler = getComponentByPropertyName(propertyName, PdqErrorHandlingService.class);
+        }
+        return handler;
+    }
+    
+    /**
+     * Get the PDQ endpoint
+     * @return
+     */
+    public String getPdqEndpoint() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PDQ_ENDPOINT, "localhost");
+    }
+
+    /**
+     * Get the PDQ port
+     * @return
+     */
+    public Integer getPdqPort() {
+        return Integer.valueOf(Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PDQ_PORT, "8989"));
+    }
+
+    public String getPixEndpoint() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PIX_ENDPOINT, "localhost");
+    }
+
+    public Integer getPixPort() {
+        return Integer.valueOf(Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PIX_PORT, "3600"));
+    }
+
+    public String getMpiUsername() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_ACCESS_USERNAME, "admin");
+    }
+
+    public String getMpiPassword() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_ACCESS_PASSWORD, "admin");
+    }
+
+    public String getShrPatientRoot() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_PATIENT_ROOT, "1.2.3.4.5.9");
+    }
+
+    public String getReceivingApplication() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_RECEIVING_APPLICATION, "CR");
+    }
+
+    public String getReceivingFacility() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_RECEIVING_FACILITY, "Clinic");
+    }
+
+    public String getSendingApplication() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_SENDING_APPLICATION, "OPENMRS");
+    }
+
+    public String getSendingFacility() {
+        return Context.getAdministrationService().getGlobalProperty(
+                RegistrationCoreConstants.GP_MPI_SENDING_FACILITY, "LOCATION");
```

Comment:
Same here, I don't think the default value make sense

Reply:
changed to "Demo Sending Facility" and made to changed config.xml to match

Reply:
Same here, I don't think you need a default value